### DISCUSSION TITLE
I don't receive benefit. please check my status.

### DISCUSSION THREAD

**Discussion by ssep4u** [community/community#129508] [GitHub Education]
### Hi everyone,

Hello, Supporter

"Approved Submitted Jun 15, 2024
Congrats! Your request for [@ssep4u](https://github.com/ssep4u) was approved and your benefits will be available [soon](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)."

but i don't receive any email and any benefits.
please, check my status.

**Comment by davevad93** [comment]
Hi @ssep4u ,

Don't worry, it may take up to a couple of weeks for the benefits to become available due to various factors, such as high volume of requests. When the benefits are available, the status will change to:

`Congrats! Your request for @ssep4u was approved and your benefits are now available`
